README.md

```diff
@@ -6,11 +6,16 @@ Feel free to contribute :smile:
 
 ## Bots
 
+* [JFreegman/rustybot](https://github.com/JFreegman/rustybot) 
+* [JFreegman/ToxBot](https://github.com/JFreegman/ToxBot/) Groupchat bot written in C.
+* [Impyy/EchoBot](https://github.com/Impyy/EchoBot)
 * [zetok/Lee](https://github.com/zetok/Lee) – inhumanly intelligent markov bot written in Rust; supports groupchats and 1v1 conversations. [<img src="https://travis-ci.org/zetok/Lee.svg?branch=master">](https://travis-ci.org/zetok/Lee)
 * [zetok/tox-neurological-warfare-bot](https://github.com/zetok/tox-neurological-warfare-bot) – spambot that delivers all the spam you desire upon request. [<img src="https://travis-ci.org/zetok/tox-neurological-warfare-bot.svg?branch=master">](https://travis-ci.org/zetok/tox-neurological-warfare-bot)
+* [zodiaclabs/CSbot](https://github.com/zodiaclabs/CSbot) (toxbot fork)
```

Comment:
could you sort those alphabetically?

I.e. not by owner of the repo, but by repo's name.